.github/workflows/e2e.yml

```diff
@@ -169,7 +169,7 @@ jobs:
   e2e-tests-notify:
     needs: [ e2e, e2e_management, e2e_s3_gcs, e2e_s3, set_neptune_ref ]
     runs-on: ubuntu-latest
-    if: (success() || failure()) && github.ref == 'refs/heads/dev/1.x'
```

Comment:
It means that notifications will be sent on even manual workflow triggers (from the Actions tab). I don't think it's expected.

Reply:
you are right. Forgot about that